ldm/simplet2i.py

```diff
@@ -158,8 +157,9 @@ def __init__(
         self.device = device
 
         # for VRAM usage statistics
-        self.session_peakmem = torch.cuda.max_memory_allocated() if self.device == 'cuda' else None
-            
+        self.session_peakmem = torch.cuda.max_memory_allocated(
+        ) if self.device == 'cuda' else None
+
```

Comment:
Ugh. I have to strenuously disagree with 'blue' on this. It makes it *harder* to understand.

Reply:
Blue did that?